web/app/services/authenticated-user.ts

```diff
@@ -77,19 +77,19 @@ export default class AuthenticatedUserService extends Service {
    */
   fetchSubscriptions = task(async () => {
     try {
-      let response = await this.fetchSvc.fetch("/api/v1/me/subscriptions", {
-        method: "GET",
-        headers: {
-          "Hermes-Google-Access-Token":
-            this.session.data.authenticated.access_token,
-        },
-      });
-      let subscriptions: string[] = await response.json();
+      let subscriptions = await this.fetchSvc
+        .fetch("/api/v1/me/subscriptions", {
+          method: "GET",
+          headers: {
+            "Hermes-Google-Access-Token":
+              this.session.data.authenticated.access_token,
+          },
+        })
```

Comment:
Not a blocker, but we shouldn't need this anymore with how we now automatically add the auth header to requests to the backend in the fetch service: https://github.com/hashicorp-forge/hermes/blob/5712506a41213cb11241375c173e53e643334594/web/app/services/fetch.ts#L13-L19

Reply:
Good catch. Addressed separately at https://github.com/hashicorp-forge/hermes/pull/91